foreshadow/tests/test_preprocessor.py

```diff
@@ -118,12 +119,13 @@ def column_summary(cls, df):
     registry._registry = _saved_registry
 
 
+@pytest.mark.skip("broken until serialization")
 def test_preprocessor_init_empty():
     """Verifies preprocessor initializes with empty values."""
 
-    from foreshadow.preprocessor import Preprocessor
+    from foreshadow.preparer import DataPreparer, ColumnSharer
```

Comment:
Oh it already is at the right domain level, then we should try to make the rest of tests consistent.